### PR TITLE
New home for Narayana LRA coordinator Docker images

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public static String getCoordinatorEndpoint() {
 
     @Override
     public Map<String, String> start() {
-        registry = new GenericContainer<>("jbosstm/lra-coordinator:5.12.0.Final")
+        registry = new GenericContainer<>("quay.io/jbosstm/lra-coordinator:latest")
```
Comment (geoand) [inline integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java:25]: Can't say I like `latest`, but let's go with it for now

**Review by geoand — APPROVED** [review]

File: integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java (hunk 1 above)
Comment (gsmet) [reply integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java:25]: Yeah, I decided to make things consistent between this and the MP tests. If it becomes a problem, we can drop it easily.

Comment (geoand) [reply integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java:25]: 👍

**Comment by Sanne** [comment]
Thanks for the quick intervention!

**Pull request merged**